File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        bool dup=false;
        int n=nums.size();
        for(int i=0;i<n;i++){
            for(int j=i+1;j<n;j++)
            if(nums[i]==nums[j])
            dup=true;
        }
        return dup;
    }
};

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
        int size=nums.length;
        int mul;
        int[] result=new int[size];
        for(int i=0;i<size;i++){
            mul=1;
            for(int j=0;j<size;j++){
                if(i==j){
                    continue;
                }
                mul=mul*nums[j];
            }
result[i]=mul;
        }

        return result;
    }
}  
